src/core/system.c

```diff
@@ -1,6 +1,7 @@
 
 #include <stdio.h>
 #include <stdlib.h>
+#include <time.h>
```

Comment:
Yes, but does it work _with_ it? It worries me that while it works flawlessly on Linux, it could be broken on any other OS. And I have no means to test it.